data/human/intro missions.txt

```diff
@@ -413,7 +413,91 @@ mission "Intro [2 Interceptor]"
+			`	"Mining can be complex compared to running jobs or trading commodities. I learned that first hand after pirates disabled my ship when I was starting out and I worked aboard a mining ship while it was being repaired. Now that was a while back so I might be a little rusty, but I could try at answering any other questions you might have about mining."`
+
+			label choices
+			choice
+				`	"What equipment do I need to mine?"`
+					goto equipment
+				`	"How do you deal with pirates?"`
+					goto pirates
+				`	"How can I check the value of minerals?"`
+					goto value
+				`	"I think I have enough information."`
+					goto end
+
+			label equipment
+			`	"Well, you need a weapon that does hull damage," James says. "A beam laser will get the job done, but it will take awhile. The most valuable tool is an asteroid scanner. They're expensive, but they connect into your navigation and targeting systems to point you towards minable asteroids. You can also set them to find the most valuable asteroids which increases your profitability per run. After that, a tactical scanner will give you range to target and show how much damage you're doing. An afterburner is good for chasing down fast moving asteroids or getting away from pirates without leaving the system. A ramscoop will let you refuel while mining, can extend your exploration range, and allow you to be ready for a quick escape."`
+			choice
+				`	"I can't fit all that on my ship!"`
+					goto shipmods
+				`	"That all makes sense."`
+					goto more
+			
+			label shipmods
+			`	James gives you a knowing smile as he answers, "Not in a factory build, you won't. But you'd be surprised how much space you can free up even in a Sparrow if you're willing to flexible about health and safety. Do you really need that much power? If you've got an outfitter on the planet where you're mining, do you need a hyperdrive, or even a shield generator? Maybe a cargo expansion would be more use." He pauses briefly, perhaps considering what liability he's opened himself up for, then adds, "I'm not saying you should, of course, but you have options.`
+				goto more
+
+			label pirates
+			`	"You can always bribe them to leave you alone. If the minerals are good enough and you have enough cargo capacity, you'll still turn a profit. Or if you have a fast ship, you can just run away. Of course, with enough of a fleet you just fight them off." He shakes his head. "Most pirates would be richer and live longer if they just mined instead of extorting bribes and cargo.`
```

Comment:
```suggestion
			`	"You can always bribe them to leave you alone. If the minerals are good enough and you have enough cargo capacity, you'll still turn a profit. Or if you have a fast ship, you can just run away. Of course, with enough of a fleet you just fight them off." He shakes his head. "Most pirates would be richer and live longer if they just mined instead of extorting merchants and plundering ships.`
```
Seems to make a little bit more sense, again probably optional.